docs/articles/nunit/writing-tests/attributes/theory.md

```diff
@@ -90,7 +90,7 @@ public class SqrtTests
         double sqrt = Math.Sqrt(num);
 
         Assert.That(sqrt >= 0.0);
-        Assert.That(sqrt * sqrt, Is.EqualTo(num).Within(0.000001));
+        Assert.That(num * num, Is.EqualTo(sqrt).Within(0.000001));
```

Comment:
This can't be right because it's asserting that `Math.Sqrt(num) == num * num` which is only true when `num` is zero.

Reply:
@jnm2 My bad, in my mind I was thinking square, not square root. Yeah, makes total sense now.

Reply:
I do know what they mean, my wires just got crossed

Reply:
No worries, we still appreciate that you said something!

Reply:
@jnm2 I was just learning how to utilise more advanced features of nunit. Well, sorry to have bothered you, now I feel like a total newbie 🤦 :.

Reply:
I've made mistakes like this too! It's no bother, trust me.